.github/workflows/release.yml

```
@@ -2,15 +2,34 @@ name: Release
 
 on:
   push:
-    tags: "*"
+    branches:
+      - master
 
 jobs:
-  npm:
+  release:
+    name: Release
     runs-on: ubuntu-latest
     steps:
-    - uses: actions/checkout@v3
-    - uses: actions/setup-node@v3
-    - run: npm ci
-    - uses: JS-DevTools/npm-publish@v1
-      with:
-        token: ${{ secrets.NPM_TOKEN }}
+      - name: Checkout Repo
+        uses: actions/checkout@v3
+        with:
+          # This makes Actions fetch all Git history so that Changesets can generate changelogs with the correct commits
+          fetch-depth: 0
+
+      - name: Setup Node.js
+        uses: actions/setup-node@v3
+        with:
+          node-version: 16
```

Let's use the latest LTS version for NodeJS:

```suggestion
          node-version: 18
```